keras/backend/tests/compute_output_spec_test.py

```diff
@@ -61,9 +61,11 @@ def single_arg_sparse_fn(x):
         ys = backend.compute_output_spec(single_arg_sparse_fn, x)
         for y in ys:
             self.assertEqual(y.shape, (None, 3, 3))
-            self.assertTrue(y.sparse)
+            # Sparseness inference is only available with TensorFlow.
```

Comment:
What would it take to make it possible with JAX?

Reply:
`compute_output_spec` is based on `jax.ShapeDtypeStruct` and `jax.make_jaxpr`, neither of them support the concept of sparse and I don't see a way to add it.
But maybe there's a way to inspect the resulting `ClosedJaxpr`? Let me look into it.

Reply:
Oh, I think I found a way to do it.

Reply:
Added.